lib/vtls/openssl.c

```diff
@@ -4570,6 +4570,72 @@ static void ossl_disassociate_connection(struct Curl_easy *data,
   }
 }
 
+static CURLcode ossl_get_tls_server_end_point(struct Curl_easy *data,
+                                              int sockindex, char **binding,
```

Comment:
```
C:\projects\curl\lib\vtls\openssl.c(4574,51): error C2220: the following warning is treated as an error [C:\projects\curl\lib\libcurl.vcxproj]
C:\projects\curl\lib\vtls\openssl.c(4574,51): warning C4100: 'sockindex': unreferenced formal parameter [C:\projects\curl\lib\libcurl.vcxproj]
C:\projects\curl\lib\vtls\openssl.c(4573,65): warning C4100: 'data': unreferenced formal parameter [C:\projects\curl\lib\libcurl.vcxproj]
```
AppVeyor is complaining about unused variables.

Reply:
I think this should only happen if `#if OPENSSL_VERSION_NUMBER > 0x10100000L` is not satisfied. I can add `(void)` casts to ignore the parameters in the `#else` branch.